core/src/main/java/ai/timefold/solver/core/api/solver/event/BestSolutionChangedEvent.java

```diff
@@ -66,6 +79,10 @@ public long getTimeMillisSpent() {
         return newBestScore;
     }
 
+    public boolean isNewBestSolutionInitialized() {
```

Comment:
Please add documentation for the new method. I couldn't find any tests using this new flag. Should we add a test case to cover it?